### ISSUE TITLE
update site down

### ISSUE THREAD

**Issue by caseyjp11** [giuspen/cherrytree#792]
https://www.giuspen.com/software/cherrytree-0.39.3.tar.xz

is not accessible. (Trying to build from Arch AUR, but also tried to just download the file. Time's out.

**Comment by giuspen** [comment]
Works for me

**Comment by caseyjp11** [comment]
Comcast is the provider. Geo Colorado. Tried again and still no joy to either direct download or build from AUR (still timing out.) (All other internet activity from here is fine.  (This is an fyi. :-) )

**Comment by giuspen** [comment]
Thanks @caseyjp11 could other users try and report if https://www.giuspen.com/software/cherrytree-0.39.3.tar.xz works for them and roughly where they are located? I can try and upload also to github

**Comment by datavectors** [comment]
Download works fine here in U.K.

**Comment by MJimitater** [comment]
+1 in Germany

**Comment by caseyjp11** [comment]
Yeah, still no go at this point. If I click on your download link, the page times out. AUR still times out and also tried a different AUR builder. They all point to the same file address, and, at least from this neck of the woods, is not working. (Other AUR apps build from other locations (including git) and net connection is good.

**Comment by giuspen** [comment]
I'm going to upload it to GitHub later

**Comment by giuspen** [comment]
Actually you can easily downlad the source code of a tag as follows: (for example 0.39.3): https://github.com/giuspen/cherrytree/archive/0.39.3.zip

**Comment by caseyjp11** [comment]
I checked the package build file and saw no issues. Replaced the normal file location with the github location and skipped integrity testing for it (it fails as the AUR hash is looking for the giuspen package rather than github). The application build downloaded and updated successfully.  From here, I still cannot download from the giuspen site, nor will the AUR package build without modifying the source location.  For now call this resolved. :-)
